### ISSUE TITLE
compile a batch for RFB annotation project

### ISSUE THREAD

**Issue by keighrim** [clamsproject/aapb-annotations#44]
A set of videos that (supposedly) have various instances of "scenes with text" (https://github.com/clamsproject/app-swt-detection/issues/1) that are ideal for creating labeled data for roles and fillers (key-value pairs) extraction and linker app. 

A few things to consider
1. consider going beyond newshour videos. 
2. consider mixing different shows and collections, while considering balance between sizes

**Comment by keighrim** [comment]
And here's are number of mp4 files we currently have; 

``` bash 
$ for d in /llc_data/clams/wgbh/* ; do if [ -d $d ]; then echo $d ; find $d -name "*.mp4" | wc -l ; fi ;done
/llc_data/clams/wgbh/credits
97
/llc_data/clams/wgbh/great_depression
9
/llc_data/clams/wgbh/NewsHour
2896
/llc_data/clams/wgbh/NJN_Network
46
/llc_data/clams/wgbh/Peabody
492
/llc_data/clams/wgbh/sonyids_connecticut_pb_20180423101623
1132
/llc_data/clams/wgbh/sonyids_newjerseynetwork_20180423192048
3152
/llc_data/clams/wgbh/thumbdrive
5
/llc_data/clams/wgbh/transcripts
0
/llc_data/clams/wgbh/usbkey
0
/llc_data/clams/wgbh/wrvr
847
```

**Comment by keighrim** [comment]
One additional (completely optional) characteristic of the video is the production/air period. The pbcore information is accessible by going to https://americanarchive.org/catalog/cpb-aacip_xxx-yyyyyyy.pbcore address.
(note where the `_` is located instead of `-`, and `.pbcore` extension in the address).

**Comment by keighrim** [comment]
fixed via #45.
